assets/javascripts/discourse/widgets/language-switcher-menu.js.es6

```diff
@@ -49,37 +48,6 @@ export default createWidget("language-switcher-menu", {
   },
 
   html() {
```

Comment:
This seems like a relatively simple widget now that the clickOutside logic is gone. Could we take a stab (like... 30 mins or less) at converting to a glimmer component? Seems a shame to upgrade to the new component-based header API and then just render a widget 😅